### ISSUE TITLE
Lock x, y or z coordinates when using PDT design operations

### ISSUE THREAD

**Issue by uporersith** [Clockmender/Precision-Drawing-Tools#28]
I would like to move vertexes in edit mode only along 1 axis at a time, for example, move only the z coordinate of a vertex while maintaining the x and y coordinates. 

Can you add a toggle to the x,y,z, coordinate input to "lock them" to each vertex's individual coordinates? 
For example, I want to move the object along the z axis to a specific global z coordinate. I would like to toggle off x and y coordinate and just input the z coordinate i want the object to moved to.

**Comment by Clockmender** [comment]
Yes, I can look at that, shouldn't be too difficult a task to achieve. Give me a little time to find the best way to do it.
